link/src/test/java/com/stripe/android/link/ui/wallet/WalletScreenTest.kt

```diff
@@ -37,7 +40,18 @@ internal class WalletScreenTest {
 
     @Test
     fun `wallet list is collapsed on start`() = runTest(dispatcher) {
-        val viewModel = createViewModel()
+        val linkAccountManager = FakeLinkAccountManager()
+        linkAccountManager.listPaymentDetailsResult = Result.success(
+            ConsumerPaymentDetails(
+                paymentDetails = listOf(
+                    TestFactory.CONSUMER_PAYMENT_DETAILS_CARD.copy(
+                        expiryYear = 2099
+                    ),
```

Comment:
can you update the expiry year in test factory instead of doing it here? I think that the default state should be that the valid is valid/not expired